mkosi

```diff
@@ -2644,7 +2644,7 @@ def find_password(args):
     try:
         require_private_file('mkosi.rootpw', 'root password')
 
-        with open('mkosi.rootpw', 'r', encoding="utf-8") as f:
```

Comment:
are you sure this is the right thing to do? i.e. dropping the encoding= bit means we read some binary strings from it, not proper unicode strings. Unless I have misunderstood something there this doesn't look right...